docs/docusaurus.config.ts

```diff
@@ -147,6 +148,16 @@ const config: Config = {
       }),
     ],
   ],
+  plugins: [
+    [
+        'docusaurus-plugin-module-alias',
+        {
+            alias: {
+                '@examples': resolve(__dirname, '../src/libecalc/examples/'),
```

Comment:
Very nice!

Reply:
FYI. I first tried to implement the webpack alias as a small custom plugin (in its own folder/files) to avoid 3rd party dependency. But that did not work. I then tried this plugin and it worked as intended.

Reply:
I just updated the snapshot for that one test since it failed after the changes. I am not sure how to validate if it is correct or not?